src/main/java/com/mitchej123/hodgepodge/core/HodgePodgeClient.java

```diff
@@ -74,4 +82,16 @@ public static int renderBlockVine_colorMultiplier(int oColor, Block block, int x
         if (type == null) return oColor;
         return type.getColor(oColor, x, z);
     }
+
+    public enum AnimationMode {
+        NoAnimation,
+        AnimateVisible,
+        AnimateAll;
+    }
+
+    public enum RenderDebugMode {
+        Off,
```

Comment:
I am not sure if that was intended but usually convention is to use UPPER_CASE for enums.

Reply:
The idea is this will be displayed to user so doesn't have to be full cap, but ok.

Reply:
You can always add a `public static final String displayName;` field to the enum